config/initializers/redis_config.rb

```diff
@@ -1,12 +1,9 @@
-rails_root = ENV['RAILS_ROOT'] || File.dirname(__FILE__) + '/../..'
-rails_env = ENV['RAILS_ENV'] || 'development'
-
-# use config/redis.yml to load settings
-redis_config_path = "#{rails_root}/config/redis.yml"
-redis_config = YAML.load(ERB.new(IO.read(redis_config_path)).result)
+require 'redis'
 
 # initialize redis connection
-Resque.redis = redis_config[rails_env]
-
-# pull out the parsed redis driver from resque
-$redis = Resque.redis.instance_eval{@redis}
+if ENV['REDIS_URL']
+  $redis = Redis.new
+else
+  # use config/secrets.yml to load settings
+  $redis = Redis.new(:url=> Rails.application.secrets.redis_url)
```

Comment:
its alphabetical order yes, so when you open the folder, the first intializer runs first. If you need to change or make sure of a certain order, you can prefix your intiailziers with numbers.

Heres an example of this in the wild: https://github.com/discourse/discourse/tree/master/config/initializers

So we okay here since redis comes before resque.